Dependencies/napi/OnLinkedAsDependency.cmake

```
@@ -1,27 +1,31 @@
 # Callback to perform custom behavior -- in this case, copying runtime output artifacts like DLLs -- when 
 # linked from an executable target as a library.
 function(on_linked_as_dependency target)
-    # We only have to do anything if the JavaScript engine is V8.
-    if (NAPI_JAVASCRIPT_ENGINE STREQUAL "V8")
-
-        # Propagate this file to the target so that it will be transitively available to targets that
-        # link to that one, too.
-        propagate_on_linked_as_dependency_cmake_file(napi ${target})
-    
+    # Propagate this file to the target so that it will be transitively available to targets that
+    # link to that one, too.
+    propagate_on_linked_as_dependency_cmake_file(napi ${target})
+    if (DEFINED NAPI_JAVASCRIPT_RUNTIME_OUTPUT_ARTIFACTS)    
```

nit: extra spaces at the end
```suggestion
    if (DEFINED NAPI_JAVASCRIPT_RUNTIME_OUTPUT_ARTIFACTS)
```